business/health.go

```diff
@@ -188,6 +191,7 @@ func (in *HealthService) getNamespaceServiceHealth(namespace string, services []
 	// Prepare all data (note that it's important to provide data for all services, even those which may not have any health, for overview cards)
 	for _, service := range services {
 		h := models.EmptyServiceHealth()
+		h.Requests.HealthAnnotations = models.GetHealthAnnotation(service.Annotations, HealthAnnotation).(map[string]string)
```

Comment:
I've commented in the "GetHealthAnnotation", it can return directly a "map[string]string" with nil or error if you want to check that the annotation is not well formed.